aspnetcore/security/enforcing-ssl.md

```diff
@@ -113,6 +129,13 @@ Per [OWASP](https://www.owasp.org/index.php/About_The_Open_Web_Application_Secur
 * The browser stores configuration for the domain that prevents sending any communication over HTTP. The browser forces all communication over HTTPS. 
```

Comment:
Above where it says " * The middleware logs a warning." Include the text of the warning: "Failed to determine the https port for redirect."